modules/user/main.tf

```
@@ -1,7 +1,8 @@
 data "opslevel_users" "all" {}
 
 resource "opslevel_user" "this" {
-  email              = var.email
+  email = var.email
+  # force_send_invite  = var.force_send_invite  # TODO: uncomment on next release
```

```suggestion
  email                       = var.email
  force_send_invite  = var.force_send_invite
```

We can stage this but just not merge it till the TF provider is released